### PR TITLE
Update URL for jhagas/ESPSupabase

### PR DESCRIPTION
**Pull request by jhagas**
Initially, this library is designed for ESP32. By the work of contributors, we make this library works for ESP8266, so i change the name of library and the GitHub link, so people will not confuse about this library support.

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @jhagas. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Comment by per1234** [comment]
```text
ERROR: Library name ESPSupabase not found in the Library Manager index. Library names are not allowed to change after   
       being added to the index.                                                                                        
       See: https://github.com/arduino/library-registry/blob/main/FAQ.md#how-can-i-change-a-librarys-name               
       (Rule LP018)                                                                                                     
```

This error from the linter is expected and will be resolved by the registered name update provided by

**Review by umbynos — APPROVED** [review]
Done in https://github.com/arduino/library-registry/pull/4805#pullrequestreview-2204912183

**Pull request merged**
